lib/coqProject_file.ml

```diff
@@ -8,7 +8,7 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
-(* This needs to go trou feedback as it is invoked from IDEs, but
+(* This needs to go trough feedback as it is invoked from IDEs, but
```

Comment:
```suggestion
(* This needs to go through feedback as it is invoked from IDEs, but
```

Reply:
*facepalm* its the second time I fixed this, but previously I didn't commit it. This time I got careless.